scripts/ingest.sh

```
@@ -79,6 +92,11 @@ while [[ $# -gt 0 ]]; do
             SKIP_HARVEST=true
             shift
             ;;
+        --mapping-only)
```

Shellcheck (0.11.0)

[warning] This pattern never matches because of a previous pattern on line 90.

(SC2222)